spec/index.emu

```diff
@@ -107,6 +107,27 @@ location: https://rawgit.com/tc39/proposal-realms/master/index.html
           </emu-note>
       </emu-clause>
 
+      <emu-clause id="sec-realm.prototype.import">
+          <h1>Realm.prototype.import ( _specifier_ )</h1>
+
+          The following steps are taken:
+
+          <emu-alg>
+          1. Let _O_ be *this* value.
+          1. Perform ? RequireInternalSlot(_O_, [[Realm]]).
+          1. Let _referencingScriptOrModule_ be *null*.
+          1. Let _specifierString_ be ? ToString(_specifier_).
+          1. Let _promiseCapability_ be ! NewPromiseCapability(%Promise%).
+          1. IfAbruptRejectPromise(_specifierString_, _promiseCapability_).
+          1. Perform ! HostImportModuleDynamically(_referencingScriptOrModule_, _specifierString_, _promiseCapability_).
```

Comment:
@erights @littledan @ljharb this is another surprising part for me, I could not find any reference to the realm in the import call specification. Is it underspecified? or am I missing something? I'm talking about https://tc39.es/ecma262/#sec-import-calls and the abstract operations called from there.

Reply:
it's probably assumed to be implicit? not sure tho.

Reply:
"assumed implicit" === underspecified. I suspect the spec has a lot of these bugs that we'll need to fix.

Reply:
HTML figures this out using the ES spec's [current realm](https://tc39.es/ecma262/#realm) concept. I imagine other hosts could do the same. That works today (there's nothing unspecified, and even `import()` within an `evaluate` in a Realm should work fine), but wouldn't work for the implementation of this method, and needs to be updated.

Reply:
@littledan let's work on that particular piece in another PR to avoid confusion.

Reply:
Thinking about this a bit more, I think the simplest way through this issue would be to add a dummy execution context stack entry to set the current realm appropriately. This can be done by cribbing lines 2-6 of [InitializeHostDefinedRealm](https://tc39.es/ecma262/#sec-initializehostdefinedrealm), and saving and restoring the surrounding calling execution context as in steps 1, 9 and 11 of [[[Call]]](https://tc39.es/ecma262/#sec-built-in-function-objects-call-thisargument-argumentslist). What would you think of applying that change as part of this patch?

Reply:
Fix proposed in https://github.com/tc39/proposal-realms/pull/233